contracts/core/tokens/rebalancing-v2/RebalancingSetState.sol

```diff
@@ -127,10 +131,7 @@ contract RebalancingSetState {
     /* ============ Modifier ============ */
 
     modifier onlyManager() {
-        require(
-            msg.sender == manager,
-            "Must be manager"
-        );
+        validateManager();
```

Comment:
Are we ever using this in a non-modifier way? Don't know why it needs to be a separate function

Reply:
It's more bytecode-efficient to separate the function out, as each piece of logic and the reason strings are duplicated across each function that utilizes this modifier

Reply:
I see, ok that makes sense